Sources/WebDriver/Session.swift

```diff
@@ -175,7 +175,7 @@ public class Session {
             return PollResult(value: elementId, success: elementId != nil)
         }.value
 
-        return elementId.map { Element(session: self, id: $0) }
+        return elementId.map { Element(session: self, id: $0, foundUsing: using, foundUsingValue: value) }
```

Comment:
The original query might also have had a `startingAt` value, so immediately rerunning the query using `foundUsing` and `foundUsingValue` would not return the same result.